crates/holochain_sqlite/src/sql/cell/schema/8-up.sql

```diff
@@ -0,0 +1,5 @@
+-- All the content in this table will be rebuilt by Kitsune2 if it is missing.
+-- Because we are fixing a logic issue with how ops are located in the DHT, clearing this table is the simplest way to
+-- ensure that when nodes upgrade, they will rebuild the SliceHash table with the correct data.
```

Comment:
Is there anything to consider here given that nodes won't update simultaneously? I guess syncing between nodes will just only start working properly once both nodes updated holochain and each calculate locations properly on their respective ends. There shouldn't be any other implications, right?

Reply:
I've tried to think this through, it's a bit of a puzzle. I think your summary is correct.

The thing I was most concerned about was indexing into op ids without checking length. I think we're safe to do that because we never trust the op ids sent to us by another peer. We always re-compute those from the op data. Once ops are stored, we also compute op ids when we look up op hashes/basis hash. 

I'm reasonably confident that this will run on existing data and the sync should start to work when multiple nodes upgrade as you say.

I think we can and should test that on an RC, because the Dino Adventure network is in a bad state too and we should see that sync up.